pkgs/top-level/config.nix

```diff
@@ -20,6 +20,11 @@ let
 
     /* Internal stuff */
 
+    # Hide built-in module system options from docs.
+    _module.args = mkOption {
+      internal = true;
```

Comment:
Do we really want this? I would like to look for nixpkgs config directly from the nixos options search site.